src/crawl4ai_mcp.py

```
@@ -90,99 +104,61 @@ def format_neo4j_error(error: Exception) -> str:
+    logger.info("Initializing MCP server components...")
+
     # Create browser configuration
-    browser_config = BrowserConfig(
-        headless=True,
-        verbose=False
-    )
+    browser_config = BrowserConfig(headless=True, verbose=False)
 
     # Initialize the crawler
     crawler = AsyncWebCrawler(config=browser_config)
     await crawler.__aenter__()
```

Pylint (3.3.7)

[convention] Unnecessarily calls dunder method __aenter__. Invoke context manager directly.

(C2801)